src/patternfly/demos/Toolbar/examples/toolbar-simple-example.hbs

```diff
@@ -0,0 +1,50 @@
+{{#> toolbar pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md"}}
+  {{#> toolbar-group}}
+    {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+    {{#> input-group }}
+      {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+        All
+      {{/dropdown}}
+       {{#> form-control controlType="input" input="true" type="text" id="textInput10" name="textInput10"  placeholder="Filter..." aria-label="filter input with popup button"}}
+      {{/form-control}}
+      {{#> button btnClass="pf-m-tertiary" btnAttributes='id="textAreaButton2"'}} <i class="fas fa-search aria-hidden="true"></i> {{/button}}
```

Comment:
can you break this into 3 lines with the `<i>` indented to be consistent?

Reply:
yes